x-pack/plugins/apm/public/components/app/service_inventory/ServiceList/index.tsx

```diff
@@ -55,186 +61,251 @@ const ToolTipWrapper = styled.span`
+      width: px(unit * 10),
+    },
+  ];
+}
+
 export function ServiceList({ items, noItemsMessage }: Props) {
   const displayHealthStatus = items.some((item) => 'healthStatus' in item);
 
+  const showTransactionTypeColumn = items.some(
+    ({ transactionType }) =>
+      transactionType !== TRANSACTION_REQUEST &&
+      transactionType !== TRANSACTION_PAGE_LOAD
+  );
+
+  const serviceColumns = useMemo(
+    () => getServiceColumns({ showTransactionTypeColumn }),
+    [showTransactionTypeColumn]
+  );
+
   const columns = displayHealthStatus
-    ? SERVICE_COLUMNS
-    : SERVICE_COLUMNS.filter((column) => column.field !== 'healthStatus');
+    ? serviceColumns
+    : serviceColumns.filter((column) => column.field !== 'healthStatus');
   const initialSortField = displayHealthStatus
     ? 'healthStatus'
     : 'transactionsPerMinute';
 
   return (
-    <ManagedTable
-      columns={columns}
-      items={items}
-      noItemsMessage={noItemsMessage}
-      initialSortField={initialSortField}
-      initialSortDirection="desc"
-      initialPageSize={50}
-      sortFn={(itemsToSort, sortField, sortDirection) => {
-        // For healthStatus, sort items by healthStatus first, then by TPM
-        return sortField === 'healthStatus'
-          ? orderBy(
-              itemsToSort,
-              [
-                (item) => {
-                  return item.healthStatus
-                    ? SERVICE_HEALTH_STATUS_ORDER.indexOf(item.healthStatus)
-                    : -1;
-                },
-                (item) => item.transactionsPerMinute?.value ?? 0,
-              ],
-              [sortDirection, sortDirection]
-            )
-          : orderBy(
-              itemsToSort,
-              (item) => {
-                switch (sortField) {
-                  // Use `?? -1` here so `undefined` will appear after/before `0`.
-                  // In the table this will make the "N/A" items always at the
-                  // bottom/top.
-                  case 'avgResponseTime':
-                    return item.avgResponseTime?.value ?? -1;
-                  case 'transactionsPerMinute':
-                    return item.transactionsPerMinute?.value ?? -1;
-                  case 'transactionErrorRate':
-                    return item.transactionErrorRate?.value ?? -1;
-                  default:
-                    return item[sortField as keyof typeof item];
+    <EuiFlexGroup direction="column" responsive={false} alignItems="flexEnd">
+      <EuiFlexItem>
+        <EuiFlexGroup responsive={false} alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              position="top"
+              content={i18n.translate(
+                'xpack.apm.servicesTable.tooltip.metricsExplanation',
+                {
+                  defaultMessage:
+                    'Service metrics are aggregated on transaction type "request", "page-load", or the top available transaction type.',
```

Comment:
Should we use 'metrics' here? Maybe stats is better? Or are we okay with using metrics in user-facing text?

Reply:
@formgeist / @bmorelli25 WDYT?

Reply:
I prefer metrics, but I'm not aware of any initiatives to avoid using metrics in user-facing text.